### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by RenSchuon**

### CODE REVIEW

**Comment by eloisab17** [comment]
Theres some broken links to the images in the animal folder
image

**Comment by eloisab17** [comment]
Also in the weather folder
image

**Comment by eloisab17** [comment]
Light is broken in the quantity folder
image

**Comment by eloisab17** [comment]
image

**Comment by eloisab17** [comment]
Light works in this folder, so the url for this icon should be used for light everywhere.
image

**Comment by eloisab17** [comment]
> Theres some broken links to the images in the animal folder image

it looks like these images are broken because the turtle, frog and snake images didnt get uploaded.

File: database/seeders/BoardSeeder.php [hunk 1]
```diff
@@ -186,14 +227,22 @@ public function run()
                     ]);
 
                 //Sensory
+                    //icons
+                        $fire_url = Storage::disk('public')->url('Icons/fire.png');
+                        $snowflake_url = Storage::disk('public')->url('Icons/snow-flake.png');
```
Comment (eloisab17) [inline database/seeders/BoardSeeder.php:232]:
```suggestion
                        $snowflake_url = Storage::disk('public')->url('Icons/snowflake.png');
```

Comment (eloisab17) [reply database/seeders/BoardSeeder.php:232]: file name typo

File: database/seeders/BoardSeeder.php [hunk 2]
```diff
@@ -283,16 +344,25 @@ public function run()
...
                 
                 //Size
+                    //icons
+                        $long_url = Storage::disk('public')->url('Icons/long.png');
+                        $tall_url = Storage::disk('public')->url('Icons/tall.png');
+                        $short_url = Storage::disk('public')->url('Icons/short.png');
+                        $big_url = Storage::disk('public')->url('Icons/big.png');
+                        $medium_url = Storage::disk('public')->url('Icons/medium.png');
+                        $small_url = Storage::disk('public')->url('Icons/small.png');
+                        $lightq_url = Storage::disk('public')->url('Icons/lightq.png');
```
Comment (eloisab17) [inline database/seeders/BoardSeeder.php:354]:
```suggestion
                        $lightq_url = Storage::disk('public')->url('Icons/light-q.png');
```

**Comment by eloisab17** [comment]
> image

looks like the attractive icon is also missing

File: database/seeders/BoardSeeder.php [hunk 3]
```diff
@@ -134,12 +161,19 @@ public function run()
...
                         ['id' => $adjectives+13, 'text' => 'Mean', 'color' => $default_color],
-                        ['id' => $adjectives+14, 'text' => 'Gay', 'color' => $default_color],
-                        ['id' => $adjectives+15, 'text' => 'Queer', 'color' => $default_color],
-                        ['id' => $adjectives+16, 'text' => 'Attractive', 'color' => $default_color],
+                        ['id' => $adjectives+14, 'text' => 'Gay', 'color' => $default_color, 'icon' => $rainbow_url],
+                        ['id' => $adjectives+15, 'text' => 'Queer', 'color' => $default_color, 'icon' => $rainbow_url],
+                        ['id' => $adjectives+16, 'text' => 'Attractive', 'color' => $default_color, 'icon' => $attractive_url],
                         ['id' => $adjectives+17, 'text' => 'Beautiful', 'color' => $default_color],
                         ['id' => $adjectives+18, 'text' => 'Handsome', 'color' => $default_color],
                         ['id' => $adjectives+19, 'text' => 'Goregous', 'color' => $default_color]
```
Comment (lvs-vong) [inline database/seeders/BoardSeeder.php:179]:
```suggestion
                        ['id' => $adjectives+19, 'text' => 'Gorgeous', 'color' => $default_color]
```
I think there is a typo here for "Gorgeous".

**Pull request merged**